### PR TITLE
Add new PublishAsync to send multiple messages in batches

### PR DESCRIPTION
**Pull request by TomMalow**
The new method will publish multiple messages in a batch. If the messages exceed the batch size, the current batch will be sent and remaining will be send in a new batch. Multiple batches can be sent 
The method will throw an exception if a single message exceeds the total allowed size of a single batch. Does not rollback any previous sent messages.

### CODE REVIEW

**Comment by christianhelle** [comment]
@TomMalow the build fails due to incorrect XML code documentation

**Review by christianhelle — APPROVED** [review]
Looks good to me

File: src/Atc.Azure.Messaging/ServiceBus/ServiceBusPublisher.cs [hunk 1]
```diff
@@ -28,6 +28,60 @@ public ServiceBusPublisher(IServiceBusSenderProvider clientProvider)
...
+        foreach (var message in messages)
+        {
+            if (cancellationToken.IsCancellationRequested)
+            {
+                break;
+            }
+
+            var busMessage = CreateServiceBusMessage(
+                    sessionId,
+                    JsonSerializer.Serialize(message),
```
Comment (christianhelle) [inline src/Atc.Azure.Messaging/ServiceBus/ServiceBusPublisher.cs:54]: I think this pull request should go in but I also noticed that we are very opinionated about how the messages are serialized

**Comment by christianhelle** [comment]
Do you feel like adding usage examples in the README @TomMalow? Otherwise, I can do that in a separate pull request

File: src/Atc.Azure.Messaging/ServiceBus/ServiceBusPublisher.cs (hunk 1 above)
Comment (TomMalow) [reply src/Atc.Azure.Messaging/ServiceBus/ServiceBusPublisher.cs:54]: I agree and have intention to separate this in a later PR

Comment (TomMalow) [reply src/Atc.Azure.Messaging/ServiceBus/ServiceBusPublisher.cs:54]: This was also the main reason why I could not use this package initially as I have customer converters

**Review by christianhelle — APPROVED** [review]
Nice!

**Pull request merged**